panther_detections/providers/zoom/__init__.py

```diff
@@ -0,0 +1,10 @@
+from typing import Literal
+
+from . import rules, sample_logs
+
+
+def use_all_with_defaults() -> None:
+    rules.user_promoted_to_privileged_role()
+    rules.operation_passcode_disabled()
+    rules.all_meetings_secured_with_one_option_disabled()
+    rules.automatic_sign_out_disabled()
```

Comment:
I know we haven't been doing this. But can we start returning a list of the content from these use-all funcs?
```suggestion
def use_all_with_defaults() -> List[Union[detection.Rules]]:
    return [
        rules.user_promoted_to_privileged_role()
        rules.operation_passcode_disabled()
        rules.all_meetings_secured_with_one_option_disabled()
        rules.automatic_sign_out_disabled()
    ]
```

Reply:
Just think it might be better if users always can access the stuff we make

Reply:
Yo @maxrichie5, what is the advantage of using `List[Union[detection.Rules]]` vs `List[detection.Rules]`? Also, I thought `Union` requires 2+ args?

Reply:
It can have just one arg. I added the `Union` part in my suggestion because these `use_all...` funcs will probably have `Query` and `Policy` and etc. in the near future. But we can add it then if we want to. No need to add it now

Reply:
Ah makes total sense, cool, we will add it once queries are converted, are policies in scope for v1 conversion or still a ways off?

Reply:
Policies are in scope :)